spec/spec.md

```diff
@@ -499,6 +490,31 @@ Below is a non-normative example of a decoded VC in a JSON format, signed as a J
 ```
 :::
 
+#### [[def:Authentication Response Validation]]
+
+The following checks MUST be made by the Verifier upon receiving the Authorization Response:
+
+##### ID Token Validation
+
+To validate the ID Token received, the Verifier MUST do the following:
+
+1. The Verifier MUST validate ID Token according to the rules defined in section 12.1 of [[ref: SIOPv2 ID1]].
+2. The ID Token MUST be signed by the End-User's DID:
+3. `iss` claim MUST be `https://self-issued.me/v2/openid-vc`.
+4. The signature on the ID Token MUST be validated. Validation is performed against the key obtained from a DID Document. DID Document MUST be obtained by resolving a Decentralized Identifier included in the `sub` claim using DID Resolution.
+5. `sub` claim MUST equal the `id` property in the DID Document.
+6. The `_vp_token` claim MUST be present and contain a presentation_submission with a valid descriptor map.
+
+##### VP Token Validation
+
+To Validate the VP Token received, the Verifier MUST do the following:
+
+1. The signature on the VP Token MUST be validated. Validation is performed against the key obtained from a DID Document. DID Document MUST be obtained by resolving a Decentralized Identifier included in the `sub` claim using DID Resolution.
```

Comment:
> VP Token that includes one or more Verifiable Presentations

VP Token can include one or more VCs - step 0 should probably be identifying the number of VPs in a VP Token using descriptor map in the ID Token, than signature on each VP gets validated.